website_sale_vat_required/tests/test_website_sale_vat_required.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import HttpCase
+
+
+class TestWebsiteSaleVatRequired(HttpCase):
+    def test_website_sale_vat_required(self):
+        if self.env["ir.module.module"]._get("payment_custom").state != "installed":
```

Comment:
Functionality tested locally, thank you!
I agree with David, the tests should be changed as this step can be avoided to test this module without relying on the payment methods module. @Abranes what do you think about making the change?